pkgs/development/python-modules/shiboken6/default.nix

```
@@ -47,14 +54,18 @@ stdenv'.mkDerivation rec {
     "-DBUILD_TESTS=OFF"
   ];
 
-  # Due to Shiboken.abi3.so being linked to libshiboken6.abi3.so.6.5 in the build tree,
+  postPatch = ''
+    substituteInPlace cmake/ShibokenHelpers.cmake --replace '#!/bin/bash' '#!''${BASH}'
```

Are the single quotes needed here? Who is expanding ${BASH} in this case, cmake?
 
```suggestion
    substituteInPlace cmake/ShibokenHelpers.cmake --replace '#!/bin/bash' "#!''${BASH}"
```

---

Yes it's a cmake var available in that file, so that way we don't have to care what platform it is running on and builds outside of the nix sandbox using our shiboken would work too

Should I add a comment to clarify that it is intentional and that it is using a CMake var?

---

I can also make it as a patch file, and I mean it is a reasonable enough change they could probably just have upstream

I have no clue how to upstream changes to Qt projects though, aside from I guess opening an issue in their JIRA or sending the request to their mailing list if they have one

---

some comment would be nice.

---

Done!